_data/sidebars/lb4_sidebar.yml

```diff
@@ -15,13 +15,13 @@ children:
   url: Installation.html
   output: 'web, pdf'
 
-- title: 'Getting started'
```

Comment:
Are we removing the Getting Started?  I kind of like it though.  
The garden path tutorial might be able to guide people through.  But the Getting Started can at least quickly show what LB can do? 

Not sure what you're planning for the tutorial.  If it's simple enough for people to "Get started", i'm good with removing the getting started.

Reply:
+1 to keeping Getting started as the page describing the process of ensure Node 8 + TypeScript is installed (does this need to be installed?), installing the CLI, and creating an app (Just show the built-in `ping controller`). A bare bones, I made a thing with LB4. 

Tutorial imo then should go into a more complex app.

Reply:
I think the Getting Started page can / should fold in the installation page.

Reply:
ah. just realize @virkt25 has the same comment! :)
see https://github.com/strongloop/loopback-next/issues/1004